### PR TITLE
added moses lake site

### PR DESCRIPTION
**Pull request by amiyasaha**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/moseslake**](https://counterspell-a0ra347po.hackclub.dev/moseslake) |
|:-:|
|<a href="https://counterspell-a0ra347po.hackclub.dev/moseslake">Screenshot of /moseslake</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/OlYOiMYzlOQ1iE3p4OchsMNbuvNXqdYkDRI5ycUfWumMpRzkpd6cj5cb814qMW0SaB1_hcQjfe_HxT5culvlIQ.png">(view full size)</a> |







Commit 7be10f29f5f7f353db86a78c050ce5970d5bc69f (https://counterspell-a0ra347po.hackclub.dev).

File: components/MosesLake/Steps.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <div className="">
+          <p className="py-10 mx-6 text-3xl uppercase retro">
+            How does a <span className="text-pink">game jam</span> work?
+          </p>
+          <div className="flex justify-center mx-12 my-6">
+            <div className="flex flex-wrap items-center justify-center gap-6 p-8 border-4 border-dashed border-pink inter">
+              <StepCard
+                img="/photos/3.png"
+                // TODO: Change `Example City` to the name of your city
+                text="Sign up for Counterspell Example city"
```
Comment (samdev-7) [inline components/MosesLake/Steps.js:16]: You forgot to swap out example city here.

File: components/MosesLake/Steps.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+              <StepCard
+                img="/photos/7.png"
+                text="Ship what you made and share it with the rest of the world!"
+              />
+            </div>
+          </div>
+          <div className="py-10 mx-6 space-y-4 text-xl uppercase retro">
+            <p>
+              {/* TODO: Change `Example City` to the name of your city */}
+              Counterspell Example City will go on for{" "}
```
Comment (samdev-7) [inline components/MosesLake/Steps.js:39]: You forgot to swap out example city here.

File: components/MosesLake/Guilds.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        </Tilt>
+      </div>
+
+      <div className="flex justify-center leading-[3rem] tracking-wider mx-6">
+        <div className="max-w-6xl px-6 py-6 mt-6 border-4 border-dashed border-pink">
+          <p className="text-3xl neuebit">
+            {/* TODO: Change `Example City` to the name of your city */}
+            You can join the guild you identify with for exclusive swag and
+            badges! You also get to compete against other guilds in friendly
+            activities/challenges throughout Counterspell Example City!
```
Comment (samdev-7) [inline components/MosesLake/Guilds.js:50]: You forgot to swap out example city here!

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost there. A few things you missed!

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**